src/main/java/com/cozyhome/onlineshop/userservice/controller/AuthController.java

```diff
@@ -67,7 +67,7 @@ public ResponseEntity<String> login(@RequestBody @Valid LoginRequest loginReques
 		}
 	}
 
-	@Operation(summary = "New user registration", description = "Registering new user and sending e-mail with a link to activate e-mail.")
+	@Operation(summary = "User registration", description = "Registers a new user and sends an email with a link to activate his account.")
```

Comment:
the same, I prefer to stay summary with the qualifying word: summary = "New user registration"

Reply:
Ok